daml-assistant/exe/DAML/Assistant.hs

```diff
@@ -17,21 +17,64 @@ import System.FilePath
 import System.Directory
 import System.Process
 import System.Exit
+import System.IO
 import Control.Exception.Safe
 import Data.Maybe
+import Control.Monad.Extra
 
 -- | Run the assistant and exit.
 main :: IO ()
-main = do
+main = displayErrors $ do
     env@Env{..} <- getDamlEnv
     sdkConfigM <- mapM readSdkConfig envSdkPath
     sdkCommandsM <- mapM (fromRightM throwIO . listSdkCommands) sdkConfigM
     userCommand <- getCommand (fromMaybe [] sdkCommandsM)
+
+    whenJust envLatestStableSdkVersion $ \latestVersion -> do
+        let isHead = maybe False isHeadVersion envSdkVersion
+            test1 = not isHead && isJust envProjectPath && envSdkVersion < Just latestVersion
```

Comment:
Is `test1` the best variable name we can come up with?

Reply:
This is a good question.